actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module ActionMailbox
+  # Generic base class for all Action Mailbox exceptions.
+  class Error < StandardError; end
+
+  if defined?(Rails)
```

Comment:
This check tells me that this would be better in its own file loaded by the railtie.

Reply:
I have gone with Kasper's API idea approach and defined the actionable errors straight in the railties. WDYT?